sdk/nodejs/src/bucketing.ts

```diff
@@ -8,8 +8,10 @@ export const importBucketingLib = async (
     { logger, options }:
     { logger?: DVCLogger, options?: DVCOptions } = {}
 ): Promise<void> => {
-    Bucketing = await instantiate()
-    startTrackingMemoryUsage(Bucketing, logger, options?.reporter)
+    if (!Bucketing) {
+        Bucketing = await instantiate()
+        startTrackingMemoryUsage(Bucketing, logger, options?.reporter)
+    }
```

Comment:
I'd usually return early to reduce indentation / brackets. But that's more of a style thing.

```suggestion
    if (Bucketing) return bucketing
    Bucketing = await instantiate()
    startTrackingMemoryUsage(Bucketing, logger, options?.reporter)
```

Reply:
Also are there any tests we should add for this?

Reply:
I like that better too. I'll make the change